Add onEventUpdate to some HP luas.  Cost in menu now reflects Rhapsody in White KI.

wrenffxi opened the pull request:
<!-- place 'x' mark between square [] brackets to affirm: -->
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

* add missing onEventUpdate to several homepoints
* players with Rhapsody in White KI will now see reduced cost in HP teleport menu [Fixes #602, Relates to #800]

wrenffxi committed:
HP teleport cost now reflects Rhapsody in White KI
Added missing onEventUpdate to some teleports

zach2good reviewed (approved):
Looks good, Travis just needs a kick 👍

wrenffxi commented:
I originally targeted release with this PR.  When I re-read Zircon's message in Discord, I switched to targeting rov.  Then I rebased and force-pushed to clean up the messy commit history that resulted from the retarget.  I'm guessing those actions bonked Travis.  Sorry!
